terraform/stack_staging/.terraform.lock.hcl

```diff
@@ -0,0 +1,24 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
+
+provider "registry.terraform.io/hashicorp/aws" {
```

Comment:
Guessing this is a new thing in the most recent version of terraform?

Reply:
Yup. https://www.terraform.io/docs/configuration/dependency-lock.html

tl;dr is that changes to your dependencies are now explicitly visible (as opposed to the version hints we used to set in `provider.tf`), so you can discuss when a dependency changes.